.claude/agents/tier-1-daily/a11y-enforcer.md

```
@@ -0,0 +1,100 @@
+
+### 4. Proper Roles
+- Buttons: `accessibilityRole="button"`
+- Links: `accessibilityRole="link"`
+- Headers: `accessibilityRole="header"`
+- Images: `accessibilityRole="image"`
+
+## Output Format
+
+```
+A11y Audit: src/components/Button.tsx
+
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)